common/src/main/java/org/entcore/common/user/UserUtils.java

```diff
@@ -345,9 +346,15 @@ private static void formatPositions(JsonObject dbResult) {
 	}
 
 	public static JsonArray mapObjectToContact(final String profile, final JsonArray shareBookmarks,
-																						 final JsonArray visible, final String acceptLanguage) {
+			final JsonArray visible, final String acceptLanguage) {
+		return mapObjectToContact(profile, shareBookmarks, visible, acceptLanguage, Optional.empty());
+	}
+
+	public static JsonArray mapObjectToContact(final String profile, final JsonArray shareBookmarks,
```

Comment:
Comme dit dans la Javadoc d'Oracle les Optional sont surtout faits pour les valeurs de retour des fonctions et pas pour les paramètres d'entrée (ou les champs)
c.f.
- https://blogs.oracle.com/javamagazine/post/12-recipes-for-using-the-optional-class-as-its-meant-to-be-used
- https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Optional.html
- https://stackoverflow.com/questions/31922866/why-should-java-8s-optional-not-be-used-in-arguments